CHANGELOG.md

```diff
@@ -49,6 +49,7 @@ Ref: https://keepachangelog.com/en/1.0.0/
 
 ### Improvements
 
+* (modules/app/29-fee) [\#1339](https://github.com/cosmos/ibc-go/pull/1339) The `RegisterCounterpartyAddress` gRPC endpoint now checks if the channel exists before registering.  
```

Comment:
Changelog entry isn't super necessary since we haven't release ics29, but guess it doesn't hurt